test/e2e/presto_prometheus_connector_up_test.go

```diff
@@ -0,0 +1,51 @@
+package e2e
+
+import (
+	"bytes"
+	"github.com/operator-framework/operator-metering/test/reportingframework"
+	"github.com/stretchr/testify/require"
+	"os"
+	"os/exec"
+	"strings"
+
+	"testing"
+)
+
+func testPrometheusConnectorWorks(t *testing.T, rf *reportingframework.ReportingFramework) {
+	t.Helper()
+	out, err := exec.Command(
+		"/usr/local/bin/kubectl",
+		"-n", rf.Namespace,
+		"get", "pods",
+		"-l", "app=presto,presto=coordinator",
+		"-o", "name",
+	).Output()
+	require.Nil(t, err, "expected querying for the presto pod name would produce no error")
+	require.NotEmpty(t, string(out), "unable to parse presto pod name")
+
+	// we know the output is going to be of the form `pod/<presto coordinator pod name>` so split by '/'
+	tmp := strings.Split(string(out), "/")[1]
+	host := strings.TrimSuffix(tmp, "\n")
+
+	cmd := exec.Command(
+		"/usr/local/bin/kubectl",
+		"-n", rf.Namespace,
+		"exec",
+		host,
+		"-c", "presto",
+		"--",
+		"presto-cli",
+		"--server", "https://presto:8080",
+		"--user", "root",
+		"--catalog", "prometheus",
+		"--schema", "default",
+		"--keystore-path", "/opt/presto/tls/keystore.pem",
+		"--execute",
+		"SELECT * FROM prometheus.default.up where timestamp > (NOW() - INTERVAL '10' second)",
+	)
+	var queryResults bytes.Buffer
+	cmd.Stderr = os.Stderr
+	cmd.Stdout = &queryResults
```

Comment:
Just saw this in the e2e logs:
```go
    --- FAIL: TestManualMeteringInstall/PrometheusConnectorWorks (403.48s)
        --- FAIL: TestManualMeteringInstall/PrometheusConnectorWorks/testPrometheusConnectorWorks (0.00s)
            require.go:765: 
                	Error Trace:	presto_prometheus_connector_up_test.go:23
                	            				main_test.go:475
                	Error:      	Expected nil, but got: &os.PathError{Op:"fork/exec", Path:"/usr/local/bin/kubectl", Err:0x2}
                	Test:       	TestManualMeteringInstall/PrometheusConnectorWorks/testPrometheusConnectorWorks
                	Messages:   	expected querying for the presto pod name would produce no error
```

I think this implies that we should be logging the `queryResults`, e.g.
```go
logrus.Debugf("Query Results: %s", string(queryResults.Bytes())
```

Reply:
Whoops, I see that this is for the first command (querying for the presto-coordinator pod name). Maybe we would want to also redirect stdout/stderr for that command like we do for the presto query command.

Reply:
Hopefully we'll see from changes in ed27074